scripts/build-nuget-packages.ps1

```diff
@@ -20,3 +20,5 @@ if (!$?) {
 if (!$?) {
     exit 1
 }
+
+[System.Media.SystemSounds]::Beep.Play()
```

Comment:
Nit - trailing new line

Reply:
D'oh, don't have the `editorconfig` VSCode plugin installed on this computer :(